addons/mail/static/src/new/activity/activity_button.js

```diff
@@ -1,4 +1,4 @@
-/** @odoo-module **/
+/** @odoo-module */
```

Comment:
should have single * at the start from module too
(double * is for js doc, this is not a js doc)

Reply:
Done